### PR TITLE
Blocks Routes. Api v1

### PR DESCRIPTION
**Pull request by Bromel777**

### CODE REVIEW

File: modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v1/models/BlockInfo.scala [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+final case class BlockInfo(
+  id: Id,
+  height: Int,
+  timestamp: Long,
+  transactionsCount: Int,
+  miner: MinerInfo,
+  size: Int,
+  difficulty: Long,
+  minerReward: Long
+)
```
Comment (oskin1) [inline modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v1/models/BlockInfo.scala:19]: Let's add `version` field to it.

Comment (Bromel777) [reply modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v1/models/BlockInfo.scala]: done

File: modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v1/services/Blocks.scala [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+      InputRepo[F, D],
+      DataInputRepo[F, D],
+      OutputRepo[F, D],
+      AssetRepo[F, D]
+    ).mapN {
+      new Live[F, D](_, _, _, _, _, _, _, _, _)(trans)
+    }
+
+  final private class Live[
+    F[_]: Sync,
```
Comment (oskin1) [inline modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v1/services/Blocks.scala:57]: We can do better and avoid Sync

Comment (oskin1) [reply modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v1/services/Blocks.scala:57]: Please refer to other services for stream compilation examples.

**Review by oskin1 — CHANGES_REQUESTED** [review]
Let's reuse unchanged models from APIv0 as much as possible.

**Pull request merged**